private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -45,7 +44,7 @@ func NewCommand(
 	flags := newFlags()
 	return &appcmd.Command{
 		Use:   name + " <source>",
-		Short: "Use a source reference to decode a binary serialized message supplied through stdin.",
+		Short: "Use a source reference to encode or decode a binary serialized message supplied through stdin.",
```

Comment:
This one is kinda hard to write. We should make it clear that the input and output can both be binary or JSON though.

```suggestion
		Short: "Use a source reference to convert a binary or JSON serialized message supplied through stdin.",
```